src/WireMock.Net/Settings/ProxyUrlReplaceSettings.cs

```diff
@@ -0,0 +1,17 @@
+namespace WireMock.Settings;
+
+/// <summary>
+/// Defines an old path param and a new path param to be replaced when proxying.
+/// </summary>
+public class ProxyUrlReplaceSettings
+{
+    /// <summary>
+    /// The old path value to be replaced by the new path value
+    /// </summary>
+    public string OldValue { get; set; }

```

Comment:
because nullable is enabled for the project, also make sure to add `= null!;` for these 2 properties